services/inc/services_dynalib.h

```diff
@@ -39,6 +39,7 @@ DYNALIB_FN(services,LED_Toggle)
 DYNALIB_FN(services,LED_Fade)
 DYNALIB_FN(services,Get_LED_Brightness)
         
+DYNALIB_FN(services,set_rgb_led_change_handler)
```

Comment:
These must go at the end, or it will change the offsets of the other dynamically linked functions. This means existing firmware that tries to call panic_ would now call set_logger_output.